ui/app/templates/components/job-page/parts/title.hbs

```diff
@@ -101,6 +101,25 @@
             @awaitingConfirmation={{this.revertTo.isRunning}}
             @onConfirm={{perform this.revertTo this.job.latestStableVersion}}
           />
+        {{else if
+          (and
+            (or (eq this.job.type "batch") (eq this.job.type "system") (eq this.job.type "sysbatch"))
+            this.job.latestVersion
+          )
+        }}
```

Comment:
Should we make sure this is a non-stopped version?

Reply:
a bit further up from here, this is in the `else` condition of an `if job.stopped` block: https://github.com/hashicorp/nomad/blob/59267a96aef6602b052b94fb101a93d27f83ab5f/ui/app/templates/components/job-page/parts/title.hbs#L76-L92

I am going to simplify it to a model-set computed property, though!